### PR TITLE
Adding Container-Related User Agents

### LINKED ISSUE #7194: User-Agent for Container-related tools
Hi there,

I'd like to contribute some new regexes to detect container runtimes (docker, buildah, and friends).

My journey started on plausible/analytics#2078, then hopped to elixir-inspector/ua_inspector#29 and I was pointed here!

How can I contribute the new regexes?

I got potential regexes ([here](https://github.com/plausible/analytics/pull/2078/files#diff-c40acb1a72fded5eb0d750d363c9bb5bfb62689566b3c513324d56971f858f50)) & samples below:

**Container Runtimes & Tools**:
* containers: `containers/5.17.0 (github.com/containers/image)`
* containerd: `containerd/v1.4.13-k3s1` & `containerd/1.5.11+azure-1`
* docker: `docker/20.10.12 go/go1.16.12 git-commit/459d0df kernel/3.10.0-1160.49.1.el7.x86_64 os/linux arch/amd64 UpstreamClient(Docker-Client/19.03.12 \(linux\))`
* cri-o: `cri-o/1.16.1 go/go1.12.17 os/linux arch/amd64`
* artifactory: `Artifactory/7.21.5 72105900`
* gcr: `go-containerregistry/v0.11.0`
* buildah: `Buildah/1.23.1`
* buildkit: `buildkit/v0.10`
* Open Build Service: `BSRPC 0.9.1`
* skopeo: `skopeo/1.7.0`

**Down here there's no potential regex yet:**

* helm: `Helm/3.9.2`
* harbor-registry-client: `harbor-registry-client`

**Bots**
* renovatebot: `RenovateBot/32.99.9`

**Libraries**
* ruby-rest-client: `rest-client/2.1.0 (linux x86_64) ruby/2.7.6p219`

All user agents were taken out logs from of a production container registry.

### PR DESCRIPTION
**Pull request by josegomezr**
### Description:

Added a list of user agents pulled logs from a production docker
registry. The new user agents are:

* artifactory
* openbuildservice
* buildah
* buildkit
* containerd
* containers
* cri-o
* docker
* gcr
* libpod
* skopeo
* helm
* harbor-registry-client

And a Security Check bot:

* RenovateBot

All user agents are referred in #7194. This closes #7194.

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Comment by josegomezr** [comment]
Had a small fail in YAML Lint (due to a missing line ending a the end of the file).

It should be good to go now! :rocket:

File: Tests/Parser/Client/fixtures/library.yml [hunk 1]
```diff
@@ -395,3 +395,99 @@
...
+    type: library
+    name: artifactory
+    version: 7.21.5
+-
+  user_agent: containers/5.17.0 (github.com/containers/image)
+  client:
+    type: library
+    name: containers
+    version: 5.17.0
+
```
Comment (sanchezzzhak) [inline Tests/Parser/Client/fixtures/library.yml:410]: delete the empty line between fixture tests.

**Comment by sanchezzzhak** [comment]
I suggest making some library names more readable. 
buildkit => BuildKit
helm    => Helm
buildah => Buildah
openbuildservice => `Open Build Service` or `open-build-service`
containerd => Containerd
artifactory => Artifactory
skopeo  => Skopeo
gcr => go-container registry  (if you look at this abbreviation in statistics, it will be difficult to guess what it is)

**Review by sanchezzzhak — DISMISSED** [review]

**Comment by sanchezzzhak** [comment]
@sgiehl can you restart the actions?
https://github.com/matomo-org/device-detector/actions

**Comment by josegomezr** [comment]
Fixed the typo on `Tests/Parser/Client/fixtures/library.yml` (there was a missing `-`). Sorry for the mess

**Review by sanchezzzhak — APPROVED** [review]

**Pull request merged**